staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/mutating/patch/smd_test.go

```diff
@@ -51,35 +51,6 @@ func TestApplyConfiguration(t *testing.T) {
 		expectedResult    runtime.Object
 		expectedErr       string
 	}{
-		{
-			name: "apply configuration add to listType=map",
-			expression: `Object{
-					spec: Object.spec{
-						template: Object.spec.template{
-							spec: Object.spec.template.spec{
-								volumes: [Object.spec.template.spec.volumes{
-									name: "y"
-								}]
-							}
-						}
-					}
-				}`,
-			gvr: deploymentGVR,
-			object: &appsv1.Deployment{Spec: appsv1.DeploymentSpec{
-				Template: corev1.PodTemplateSpec{
-					Spec: corev1.PodSpec{
-						Volumes: []corev1.Volume{{Name: "x"}},
-					},
-				},
-			}},
-			expectedResult: &appsv1.Deployment{Spec: appsv1.DeploymentSpec{
-				Template: corev1.PodTemplateSpec{
-					Spec: corev1.PodSpec{
-						Volumes: []corev1.Volume{{Name: "x"}, {Name: "y"}},
-					},
-				},
-			}},
-		},
```

Comment:
What is this test case duplicated to?

Reply:
It's directly below. I think this happened during a rebase.